src/main/java/Ui.java

```diff
@@ -0,0 +1,38 @@
+public class Ui {
+    private final String logo = " ____        _        \n"

```

Comment:
```suggestion
    private final String LOGO = " ____        _        \n"

```

Capitalised since it is a constant